CONTRIBUTING.md

```diff
@@ -44,6 +65,8 @@ Please use GitHub Discussions to submit feedback and engage with community [http
 - **Search First**: It helps to check if your problem or feature request has already been discussed before opening a new issue.
 - **Be Detailed**: When you open a new issue, providing as much detail as possible really helps. Feel free to use our templates for bugs and feature requests.
 - **Be Respectful**: Let's all be kind and considerate in our communication.
+- **Solving Issues**: Limit yourself to solving a maximum of four `good first issues`. Once you've reached this limit, consider tackling other types of issues.
+- Work on only one issue at a time.
```

Comment:
- Solving issues
  - Limit ...
  - Please work on ...
  - Please ask for assignee before working
  - If there's no update for about a week on a particular issue, we'll remove the assignee

Reply:
Move this to a dedicated heading `Solving Issues`  and rename the current heading to `Creating Issues`